en/manual/requirements/index.md

```diff
@@ -7,28 +7,29 @@ To develop projects with Stride, you need:
 | Requirement     | Specifications 
 |-----------------|----------------
 |Hard drive space | 5GB
-| Operating system | Windows 10, 11
+| Operating system | Windows 10, 11 ^(1)^
```

Comment:
Is that tag actually supported ? There are no previous occurrences within the repo

Reply:
It is as far as general [Markdown](https://www.markdownguide.org/extended-syntax/#superscript) but I see that the Github MD renderer does not.  *sigh*

Since I don't know what the webserver rendering the pages can support, let me update.